kolibri/plugins/coach/assets/src/views/quizzes/CreateExamPage/sidePanels/QuizResourceSelection/index.vue

```diff
@@ -0,0 +1,431 @@
+<template>
```

Comment:
Even though github shows this as a new file, I managed to keep the blame, as this was renamed and refactored from https://github.com/learningequality/kolibri/pull/13043/files#diff-de1998b52a97bdcd29e801e6fee5e43980b32478bf3afda44537e61aba2b47a8

![image](https://github.com/user-attachments/assets/f7d6da95-a0d7-4310-9e47-a761c68d19a8)